<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoredomBreaker</class>
 <widget class="QMainWindow" name="BoredomBreaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>263</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>529</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BB</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbLoad">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>90</y>
      <width>37</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSave">
    <property name="geometry">
     <rect>
      <x>487</x>
      <y>90</y>
      <width>37</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDongle">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt; </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbDongle">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Dongle</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbWinList">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="barHP">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>175</y>
      <width>93</width>
      <height>5</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="barCP">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>93</width>
      <height>5</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="barMP">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>93</width>
      <height>5</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="barMobHP">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>198</y>
      <width>93</width>
      <height>5</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="barMobMP">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>203</y>
      <width>93</width>
      <height>5</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="barVP">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>185</y>
      <width>93</width>
      <height>5</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="chkbox_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>421</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QLabel" name="key_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>411</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_ellipsed_time">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>237</y>
      <width>93</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>waiting...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEnumerate">
    <property name="geometry">
     <rect>
      <x>503</x>
      <y>60</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFindBars">
    <property name="geometry">
     <rect>
      <x>484</x>
      <y>60</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbCondSetList">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>93</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAdd">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>22</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbConfFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="leNic">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>93</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSaveProject">
    <property name="geometry">
     <rect>
      <x>486</x>
      <y>10</y>
      <width>37</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLoadProject">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>37</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbCtrl">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>207</y>
      <width>40</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ctrl</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbShift">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>207</y>
      <width>45</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Shift</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>16</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="chkbox_widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>51</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QLabel" name="key_label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>51</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbStar">
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>190</y>
      <width>8</width>
      <height>8</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTargetType">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>151</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Unkown</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbJumpToBootloader">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="barPet1HP">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>215</y>
      <width>93</width>
      <height>5</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="barPet2HP">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>93</width>
      <height>5</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
